client/src/components/cardComponent/card.js

```diff
@@ -6,29 +6,28 @@ export default class Card extends React.Component {
     clicked: false, // when clicked it should change color
     classIs: "card cardUnclicked"
   };
-  colorClick = () => {
+  changeColorOnClick = () => {
     // confirm message first
+    let confirmValue = window.confirm(
+      `${this.state.clicked ? "Unmark" : "Mark"} ${this.props.name} ?`
+    );
+    if (!confirmValue) return;
+    fetch(`/idVoter/:${this.props.id}/status/:${!this.state.clicked}`)
+      .then(
+        this.setState({
+          clicked: !this.state.clicked,
+          classIs: this.state.clicked
+            ? "card cardUnclicked"
+            : "card cardClicked"
+        })
+      )
 
```

Comment:
Where is this component being rendered? I don't see it when I run the app and inspect it with dev tools. There seems to be a bug in one of the functions:

![screenshot 2018-09-17 14 12 14](https://user-images.githubusercontent.com/24983802/45620197-bd595d80-ba84-11e8-877d-8458e66b900b.jpg)

If it's a simple fix it can be done in this PR but if not it might be worth opening a separate issue for this.

Reply:
There is also a problem with one of the rules in .eslintrc and it's causing linting errors on the frontend:

![screenshot 2018-09-17 14 15 34](https://user-images.githubusercontent.com/24983802/45620281-06111680-ba85-11e8-8a43-12075a045799.jpg)

Reply:
#28  i opened issue for this ,
i tried to fix this by adding eslint-plugin-react,
seems not working yet.

Reply:
the above bug i think it because we are not running back and front end servers together with a script,
we are running the back end server on localhost 3000
while front end we are using 3001.
we will use the package you suggested `concurrently` it will save us a lot time  :smile: 
i will open issue for this

Reply:
I will try ut and let you know , thanks :-1: